lib/java/com/google/android/material/badge/BadgeUtils.java

```diff
@@ -47,7 +47,7 @@
 @ExperimentalBadgeUtils
 public class BadgeUtils {
 
-  public static final boolean USE_COMPAT_PARENT = VERSION.SDK_INT < VERSION_CODES.JELLY_BEAN_MR2;
+  public static final boolean USE_COMPAT_PARENT = false;
```

Comment:
I didn't know how to proceed here: `BadgeUtils` is a public class, and so is this `USE_COMPAT_PARENT` constant.
Can I simply remove it? Or should I deprecate it for removal in a future release?